### PR TITLE
#81 Use generic cmake generators

### LINKED ISSUE #81: Add DYNAWO_CMAKE_GENERATOR to build with Ninja

### PR DESCRIPTION
**Pull request by lecourtoisflo**
closes #81

### CODE REVIEW

File: util/envDynawoAlgorithms.sh [hunk 1]
```diff
@@ -353,7 +354,7 @@ config_dynawo_algorithms() {
     -DLIBZIP_HOME=$DYNAWO_LIBZIP_HOME \
     -DCMAKE_MODULE_PATH=$DYNAWO_HOME/share/cmake \
     -DDYNAWO_PYTHON_COMMAND="$DYNAWO_ALGORITHMS_PYTHON_COMMAND" \
-    -G "Unix Makefiles" \
+    -G $DYNAWO_CMAKE_GENERATOR \
```
Comment (gautierbureau) [inline util/envDynawoAlgorithms.sh:357]: "$DYNAWO_CMAKE_GENERATOR"

Comment (gautierbureau) [inline util/envDynawoAlgorithms.sh:357]: "$DYNAWO_CMAKE_GENERATOR"

**Review by gautierbureau — CHANGES_REQUESTED** [review]

File: util/envDynawoAlgorithms.sh (hunk 1 above)
Comment (lecourtoisflo) [reply util/envDynawoAlgorithms.sh:357]: quotes are already in the variable when set

Comment (gautierbureau) [reply util/envDynawoAlgorithms.sh:357]: yes but you need to add them here to to avoid the issue with the space

Comment (lecourtoisflo) [reply util/envDynawoAlgorithms.sh:357]: When put it that way, there is no problem with "Unix Makefiles"

Comment (gautierbureau) [reply util/envDynawoAlgorithms.sh:357]: Problem or not it is mandatory in bash to quotes variables and a bad practice not to quote them

**Review by gautierbureau — APPROVED** [review]

**Pull request merged**